common/src/models/service.rs

```diff
@@ -39,10 +40,10 @@ URI:           {}
 "#,
                 self.name.clone().bold(),
                 deployment.id,
-                deployment
-                    .state
-                    .to_string()
-                    .with(deployment.state.get_color()),
+                deployment.state.to_string().with(
+                    // Unwrap is safe because Color::from_str returns the color white if str is not a Color.
```

Comment:
nit: Should we have the same note above?

Reply:
Hmm, what do you mean by above?